controlplane/kubeadm/internal/workload_cluster_test.go

```diff
@@ -236,6 +234,8 @@ func TestUpdateKubeProxyImageInfo(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
+			g := NewWithT(t)
```

Comment:
Shadowing, consider renaming

Reply:
thanks so much for your review @JoelSpeed 
did all the changes you suggested